runbotics/runbotics-orchestrator-ui/src/src-app/components/InfoPanel/ProcessInstanceEventsDetails/ProcessInstanceEventsDetailsHeader.tsx

```diff
@@ -28,7 +28,10 @@ const ProcessInstanceEventsDetailsHeader: VFC<Props> = ({ processInstanceEvent }
     const { translate } = useTranslations();
     const formattedStatus = capitalizeFirstLetter({ text: processInstanceEvent.status, lowerCaseRest: true, delimiter: /_| / });
     const translateKey = `Process.Details.Modeler.Actions.${capitalizeFirstLetter({ text: processInstanceEvent.step, delimiter: '.', join: '.'})}.Label`;
-    
+    const timeDiff = 
+    formatTimeDiff(processInstanceEvent.created, processInstanceEvent.finished) ? 
+        formatTimeDiff(processInstanceEvent.created, processInstanceEvent.finished) :
+        `< 500${translate('Common.Time.Miliseconds.Short')}`;
```

Comment:
assign `formatTimeDiff(processInstanceEvent.created, processInstanceEvent.finished)` to the const above and use it here. Shorter + fn called once